src/Agoda.IoC.Core/ContainerRegistrationExceptionContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Agoda.IoC.Core
+{
+    public class ContainerRegistrationContextException : Exception
+    {
+        public ContainerRegistrationContextException(string message): base(message)
+        {
+        }
+        public bool Ignore { get; set; }
```

Comment:
I don't see any place that would set it to `true`. Why do we need it?

Reply:
I think in some cases application can run so I do need to provide this option to not break the application. what do you think